app.py

```diff
@@ -48,25 +48,24 @@ def merge_pdfs(slug):
 def document(ocd_id):
 
     client = boto3.client('s3')
-    document = client.get_object(Bucket=S3_BUCKET,
-                                 Key='{}.pdf'.format(ocd_id))
-
-    # This closure just iterates the streaming response from S3 into a
-    # streaming response from this route. That way we don't have to read
-    # anything into memory
-    def generate_response(doc):
-        while True:
-            yield doc.read(4096)
 
     try:
+        document = client.get_object(Bucket=S3_BUCKET,
+                                     Key='{}.pdf'.format(ocd_id))
+    except botocore.exceptions.ClientError as e:
```

Comment:
You don't use `e` so why do you have this formulation?

Reply:
True, @fgregg. I changed this.